.github/dependabot.yml

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2023 spdx contributors
```

Comment:
This file looks like it may have been copied from dependabot (due to the comments below), so I am not sure if this license and copyright is correct.